hazelcast-jet-core/src/main/java/com/hazelcast/jet/DAG.java

```diff
@@ -229,7 +230,19 @@ public Vertex getVertex(String vertexName) {
 
     @Override
     public String toString() {
-        return "Vertices " + verticesByName + "\nEdges " + edges;
+        final StringBuilder b = new StringBuilder("dag\n");
+        for (Iterator<Vertex> it = iterator(); it.hasNext();) {
+            final Vertex v = it.next();
+            b.append("    .vertex(").append(v.getName());
+            if (v.getLocalParallelism() != -1) {
+                b.append(".localParallelism(").append(v.getLocalParallelism()).append(')');
+            }
```

Comment:
This outputs:

    .vertex(vertexName.localParallelism(1))

Better would be:

    .vertex(vertexName)
        .localParallelism(1)

Next, I would prefer the vertexName to be in quotes.